libpod/reset.go

```diff
@@ -124,7 +124,20 @@ func (r *Runtime) Reset(ctx context.Context) error {
 	if storageConfPath, err := storage.DefaultConfigFile(rootless.IsRootless()); err == nil {
 		if _, err = os.Stat(storageConfPath); err == nil {
 			fmt.Printf("A storage.conf file exists at %s\n", storageConfPath)
-			fmt.Println("You should remove this file if you did not modify the configuration.")
+			if removeConf {
```

Comment:
This code should go into storage.conf, and you should just be editing the TOML file.  Not removing and creating one fresh.